forest/jasmine/traj2stats.py

```diff
@@ -182,11 +186,11 @@ def get_nearby_locations(
         requests.exceptions.HTTPError,
         requests.exceptions.ReadTimeout
     ) as err:
-        sys.stdout.write(f"Timeout error: {err}\n")
-        sys.stdout.write(
+        logger.error(f"Timeout error: {err}")
+        logger.error(
             "OpenStreetMap query is too large. "
             "Do not use save_osm_log or places_of_interest "
-            "unless you need them.\n"
+            "unless you need them."
```

Comment:
I would combine these two statements similar to the example in `forest/bonsai/simulate_gps_data.py`